io.github.aerocyber.sitemarker.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "io.github.aerocyber.sitemarker",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "sitemarker",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+	"--filesystem=home",
+        "--filesystem=~/.sitemarker:create"
```

Comment:
there is redundancy here. Also do you really need home? Doesn't the app use standard location for directories? (isn't it a Gtk app?)

Reply:
Do you mean that only `--filesystem=~/.sitemarker:create" is needed without `--filesystem=home`? Just making sure.

Reply:
so it probably use the standard XDG location, and portal, which mean that these permissions are excessive.

Reply:
I'm sorry, I should have mentioned earlier that the app has a hardcoded path to `~/.sitemarker`. So, would removing `--filesystem=home` be enough or should I change the upstream?